Engine/src/main/java/dk/gruppesex/bodtrd/managers/GameInputManager.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package dk.gruppesex.bodtrd.managers;
+
+import com.badlogic.gdx.Input.Keys;
+import com.badlogic.gdx.InputAdapter;
+import dk.gruppesex.bodtrd.common.data.Action;
+import dk.gruppesex.bodtrd.common.data.GameData;
+
+/**
+ *
+ * @author Morten
+ */
+public class GameInputManager extends InputAdapter
+{
+    private final GameData gameData;
+
+    public GameInputManager(GameData gameData)
+    {
+        this.gameData = gameData;
+    }
+
+    @Override
+    public boolean keyDown(int k)
+    {
+        if (k == Keys.UP)
+        {
+            gameData.getKeys().setAction(Action.MOVE_UP, true);
```

Comment:
this is unnecessary. ActionHandler is a static resource and thus you can directly call ActionHandler.setAction